### DISCUSSION TITLE
Accidental Deletion of Organization

### DISCUSSION THREAD

**Discussion by leonasdev** [community/community#101931] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Today, while managing Github Organizations, I accidentally deleted an organization instead of a repository when attempting to delete it! Is there any way you can assist me in recovering this?

I have already opened a ticket at https://support.github.com/ and wanted to inquire if anyone has had a similar experience and if their data was eventually recovered.

Additionally, the plan for our organization is GitHub Free....

**Comment by amankr1469** [comment]
On GitHub.com, deleted organizations cannot be directly recovered. If you're using GitHub Enterprise Server, things are different! Site admins can potentially restore deleted organizations. This process involves searching for the organization in the "Deleted accounts" section of the site admin dashboard and clicking "Recreate.". Restoring the organization doesn't automatically restore its repositories. You'll need to manually restore them separately.

**Reply by leonasdev** [reply]
I'm not using GitHub Enterprise Server. Is there any way to recover deleted private organization with GitHub Free?

**Reply by amankr1469** [reply]
As far ik, No. You can still try contacting GitHub support at support@github.com. They might be able to help in specific situations, but there's no guarantee, and response times can vary.
You can try creating a new organization with the same name after 90 days period.

**Reply by leonasdev** [reply]
Okay, I have open a ticket for this. Hope they can help me... still no response

**Reply by leonasdev** [reply]
To others facing this issue, I am delighted to inform you that our organization has been successfully restored. All you need to do is open a ticket on support.github.com, and the GitHub support team will kindly assist you.

In my case, aside from re-inviting members and a few basic settings (such as the profile avatar), all repositories and related PRs and issues were successfully recovered.

Special thanks to the GitHub support team for their assistance.

**Reply by JupyterCode** [reply]
> To others facing this issue, I am delighted to inform you that our organization has been successfully restored. All you need to do is open a ticket on support.github.com, and the GitHub support team will kindly assist you.
> 
> In my case, aside from re-inviting members and a few basic settings (such as the profile avatar), all repositories and related PRs and issues were successfully recovered.
> 
> Special thanks to the GitHub support team for their assistance.

 It took a few days for the problem to be resolved. 
 Are the stars of Repo back?

**Reply by ahiyantra** [reply]
@leonasdev We face the same issue.
@amankr1469 Our e-mails were blocked.
Please help us recover our deleted org (vpryzyng) on github. We submitted two support tickets over a week ago but nobody responded. Our projects are being delayed.

**Comment by Aryan-246** [comment]
Hello @leonasdev,

Accidentally deleting a GitHub organization is a serious situation, and unfortunately, there’s no self-service way to restore it once it has been deleted. Unlike repositories (which may sometimes be recoverable), organization deletion is generally considered permanent on the platform.

That said, since you’ve already contacted support, here’s the reality—no fluff:

* GitHub **Support is your only real shot.** There is no community workaround, no hidden recovery tool.
* In rare cases, GitHub staff *may* be able to restore data if the deletion is very recent and backups are still available—but this is **not guaranteed**, especially on the Free plan.
* The faster you act, the better your chances. If it's been too long, recovery becomes extremely unlikely.

What you should do right now:

1. Reply to your existing support ticket and clearly state:

   * Organization name
   * Exact deletion time (or as close as possible)
   * That it was accidental
2. Ask explicitly if a **backend restore from backups** is still possible.
3. Do **not** create multiple tickets—it slows things down.

Hard truth:
If GitHub says no, the data is gone. There’s no fallback, no recycle bin for organizations.

Going forward (learn from this, seriously):

* Never perform destructive actions without double-checking the target.
* Maintain **external backups** (repos, configs, secrets).
* Use role-based access carefully—limit who can delete organizations.

Hope support comes through for you—but be prepared for the possibility that it won’t.

Best regards.
